src/main/java/Duke.java

```diff
@@ -1,10 +1,215 @@
+import java.util.*;
```

Comment:
Perhaps it would be better to list out the specific imports that you need from java.util